src/t_string.c

```diff
@@ -209,10 +303,119 @@ int getGenericCommand(client *c) {
     return C_OK;
 }
 
+int getexGenericCommand(client *c, int flags, robj *expire, int unit) {
```

Comment:
Do we this abstraction? Seems unnecessary since there will probably only be one getex command.

Reply:
Removed it